infrastructure/cdn-in-a-box/optional/grafana/Dockerfile

```diff
@@ -21,7 +21,7 @@ USER root
 ARG TRAFFIC_TS_RPM=traffic_stats/traffic_stats.rpm
 
 RUN apt-get update && \
-    apt-get install -y dnsutils net-tools gettext-base p7zip-full netcat && \
+    apt-get --no-install-recommends install -y dnsutils net-tools gettext-base p7zip-full netcat && \
```

Comment:
See above—I would prefer disabling `APT::Install-Recommends` and `APT::Install-Suggests` in the APT configuration to passing `--no-install-recommends` for every apt or apt-get command.